src/pages/accomodation/Accommodation.tsx

```diff
@@ -94,6 +108,11 @@ function Accommodation() {
               onError={handleErrorInImage}
             />
           </Grid>
+          <ShowPhotos
+            id={accommodationId as string}
+            open={openDialog}
+            onClose={() => setOpenDialog(false)}
```

Comment:
create handleOpenDialog, otherwise it contradictory, if you have created function to open dialog, but not to close